lua/textcase/plugin/conversion.lua

```diff
@@ -63,12 +65,54 @@ function M.do_block_substitution(start_row, start_col, end_row, end_col, method)
 end
 
 function M.do_lsp_rename(method)
+  local did_apply_lsp_rename = false
+
+  local lsp_clients = vim.lsp.get_active_clients()
+
+  local lsp_handler_rename = vim.lsp.handlers["textDocument/rename"]
+  local handleLSPRenameFinished = function(applied_lsp_rename, reason)
+    reason = reason or "LSP rename failed. Verify attached Language Servers support it."
+    if not applied_lsp_rename then
+      vim.api.nvim_err_writeln(reason)
+    end
+  end
+
   local current_word = vim.fn.expand("<cword>")
   local params = lsp.util.make_position_params()
   params.newName = method(current_word)
-  lsp.buf_request(0, "textDocument/rename", params, function(err, result, context, config)
-    vim.lsp.handlers["textDocument/rename"](nil, result, context)
-  end)
+  if flag_buf_request_all then
+    lsp.buf_request_all(0, "textDocument/rename", params, function(results)
+      for _, res in pairs(results or {}) do
+        if res.result and vim.tbl_count(res.result.changes) > 0 then
+          -- TODO: Call default handler
+          -- lsp_handler_rename(res.result.err, result, context)
```

Comment:
This TODO is merged on purpose, I'd like to test the nightly badge in the readme when there is an upcoming unsupported feature merged on main